crates/viewer/re_renderer/src/importer/cpu_model.rs

```diff
@@ -0,0 +1,92 @@
+use std::sync::Arc;
+
+use slotmap::{SecondaryMap, SlotMap};
+
+use crate::{
+    mesh::{CpuMesh, GpuMesh, MeshError},
+    renderer::MeshInstance,
+    RenderContext,
+};
+
+slotmap::new_key_type! {
+    /// Key for identifying a cpu mesh in a model.
+    pub struct CpuModelMeshKey;
+}
+
+/// Like [`MeshInstance`], but for CPU sided usage in a [`CpuModel`] only.
+pub struct CpuMeshInstance {
+    pub mesh: CpuModelMeshKey,
+    pub world_from_mesh: glam::Affine3A,
+    // TODO(andreas): Expose other properties we have on [`MeshInstance`].
+}
+
+/// A model as stored on the CPU.
```

Comment:
What is the difference between a model and a mesh?

How does `CpuModel` relate to `CpuMesh`?
Is there a `GpuModel` ?